apps/console/src/features/server-configurations/components/governance-connectors/dynamic-governance-connector.tsx

```diff
@@ -31,7 +33,8 @@ import { serverConfigurationConfig } from "../../../../extensions";
 import { updateGovernanceConnector } from "../../api";
 import { getGovernanceConnectorIllustrations } from "../../configs";
 import { ServerConfigurationsConstants } from "../../constants";
-import { GovernanceConnectorInterface } from "../../models";
+import { ConnectorPropertyInterface, GovernanceConnectorInterface, UpdateGovernanceConnectorConfigPropertyInterface } 
```

Comment:
Shall we format like this since this is lengthy? Check the other places also.
```suggestion
import {
    ConnectorPropertyInterface,
    GovernanceConnectorInterface,
    UpdateGovernanceConnectorConfigPropertyInterface
} from "../../models";
```

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/3982/commits/8fd831d8584b09497fbab6218a7b567d712d87df